app/v1/__init__.py

```diff
@@ -0,0 +1,102 @@
+
+from flask_api import FlaskAPI
+from flask import request, jsonify, abort
+
+
+# local import configs and routes(route file merged here)
+from instance.config import app_config
+
```

Comment:
Why all this space here?

Reply:
Let me Google how to use your code to remove extra spacing... seems like I did not do it properly.